contact-page.css

```diff
@@ -64,18 +57,15 @@ h2 {
 }
 
 .form-box {
-    top: 10px;
-    margin: 50px 200px;
-    justify-items: center;
+    padding: 1rem 2rem;
 }
 
-.form-box #error-msg {
+.form-box .error-msg {

```

Comment:
You don't need to over-specify your CSS code. Is `.error-msg` somewhere else on the page too? If it is, is it different from this? If your answer is no, then `.error-msg` is enough to apply these rules.